lib/senkyoshi/models/qti.rb

```diff
@@ -0,0 +1,167 @@
+require "senkyoshi/models/assignment_group"
+require "senkyoshi/models/assignment"
+require "senkyoshi/models/question"
+require "senkyoshi/models/resource"
+
+QTI_TYPE = {
+  "Test" => "Assessment",
+  "Survey" => "Survey",
+  "Pool" => "QuestionBank",
+}.freeze
+
+module Senkyoshi
+  class QTI < Resource
+    def self.from(data, pre_data)
+      type = data.at("bbmd_assessmenttype").text
+      qti_class = Senkyoshi.const_get QTI_TYPE[type]
+      qti = qti_class.new
+      qti.iterate_xml(data, pre_data)
+    end
+
+    def initialize
+      @title = ""
+      @description = ""
+      @quiz_type = ""
+      @points_possible = 0
+      @items = []
+      @group_name = ""
+      @workflow_state = "available"
```

Comment:
This actually needs to stay as `published` because it is used on the assignment which uses `published` instead of `available`